From: Carlos Maiolino <cmaiolino@redhat.com>
Date: Wed, 10 Jul 2013 15:14:40 +0000 (-0300)
Subject: mkfs: add missing noalign suboption information
X-Git-Url: http://oss.sgi.com/cgi-bin/gitweb.cgi?p=xfs%2Fcmds%2Fxfsprogs.git;a=commitdiff_plain;h=1a78b0840afcfbe01c12e6af04659388d62efeea

mkfs: add missing noalign suboption information

Adds to the mkfs man page, information about noalign mkfs option which is
missing until now.

Changelog:

	V2:
	   -rephase noalign data-section option
	   -add noalign realtime device option

Signed-off-by: Carlos Maiolino <cmaiolino@redhat.com>
Reviewed-by: Eric Sandeen <sandeen@redhat.com>
Signed-off-by: Ben Myers <bpm@sgi.com>
---

diff --git a/man/man8/mkfs.xfs.8 b/man/man8/mkfs.xfs.8
index 04c19b4..61122f0 100644
--- a/man/man8/mkfs.xfs.8
+++ b/man/man8/mkfs.xfs.8
@@ -244,6 +244,11 @@ will automatically query the logical volume for appropriate
 and
 .B swidth
 values.
+.TP
+.BI noalign
+This option disables automatic geometry detection and creates the filesystem
+without stripe geometry alignment even if the underlying storage device provides
+this information.
 .RE
 .TP
 .B \-f
@@ -695,6 +700,10 @@ This is used to specify the size of the real-time section.
 This suboption is only needed if the real-time section of the
 filesystem should occupy less space than the size of the partition
 or logical volume containing the section.
+.TP
+.BI noalign
+This option disables stripe size detection, enforcing a realtime device with no
+stripe geometry.
 .RE
 .TP
 .BI \-s " sector_size"

